### ISSUE TITLE
Add support for custom TODO format per project

### ISSUE THREAD

**Issue by preslavmihaylov** [preslavmihaylov/todocheck#53]
Create a new section in the `.todocheck.yaml` config which enables one to specify a per-project custom TODO format.

E.g.
```
custom_todos:
 - FIXME
 - @todo
 - @fixme
```

This will allow users to integrate todocheck with other kinds of linters which demand a more specific todo format.

In case one specifies a TODO which matches one of this patterns but not its casing (e.g. `FiXmE`), that TODO should be detected as invalid.

Include test scenarios for this new use case as well as the edge cases mentioned above.

Finally, document the new feature in the `README` or create a follow-up task for that.

**Comment by preslavmihaylov** [comment]
Mentioned in #48

**Comment by preslavmihaylov** [comment]
@addiks FYI, this feature is now live.

Feel free to pull latest master & `go build` or `go install github.com/preslavmihaylov/todocheck` 🎉
